### DISCUSSION TITLE
Files deleted on Mountain Duck not showing in Mac Bin

### DISCUSSION THREAD

**Discussion by eyemack** [iterate-ch/cyberduck#16628] [Q&A]
When I delete a file from my OneDrive mounted by Mountain Duck, it does not go to or show in Mac Bin.  Is this normal?  Where is it going?

**Comment by dkocher** [comment]
Files deleted are moved to the recycle bin in OneDrive. You can restore these from the _Recycle bin_ item on [onedrive.live.com](https://onedrive.live.com/).

**Comment by eyemack** [comment]
Thanks, but that doesn't alter the fact that the files don't go to the Bin when using Mountain Duck.  Using the OneDrive app, they do.
I'm afraid that and the fact that MD files don't appear in Spotlight search means I'll be going back to the OneDrive app.

**Reply by dkocher** [reply]
We will have a connect mode using FileProvider API on macOS in upcoming version 5 that will show the same behaviour as the OneDrive application in this regard.

**Reply by eyemack** [reply]
Sounds good. Will that mean MD files will appear in both the Spotlight Search and the Mac Bin?
Will v5 be a free update?  I've literally just bought v4 and - for the moment - have gone back to using the OneDrive app.

**Reply by tatze96** [reply]
>Will that mean MD files will appear in both the Spotlight Search and the Mac Bin?

Correct. Within Mountain Duck 5, while using the new connection mode, Spotlight Search and the Mac Bin for deleted files will be supported.
